patches/README.md

```diff
@@ -0,0 +1,13 @@
+# Patches
+
+This directory contains patches for [patch-package](https://github.com/ds300/patch-package#readme)
+
+If you add a new patch you _must_ describe what the patch does and why it's needed in this file.
```

Comment:
👍

Reply:
cc @ds300 I'd like to build something to poke people to do this.